client/src/domain/Community/Community.css

```diff
@@ -0,0 +1,198 @@
+main section.Community {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  /* padding-bottom: 2rem; */
+
+  width: 100%;
+
+  position: relative;
+
+  overflow: hidden;
+
+  box-shadow: rgb(50 50 93 / 25%) 0px 13px 27px -5px,
+    rgb(0 0 0 / 30%) 0px 8px 16px -8px;
+  border-radius: 10px;
+  background-color: var(--card-bg-color);
+
```

Comment:
Why this empty lines here?

Reply:
Hm ... we would need to go really deep deep down (like in: childhood stuff!) to get to the bottom of that - so it's better if I just remove the lines  ... ;-)